common/views/components/EventPromo/EventPromo.js

```diff
@@ -75,6 +89,7 @@ const EventPromo = ({
             {event.title}
           </Space>
 
+          {(event.isOnline && !event.availableOnline) && <OnlineIndicator>Online</OnlineIndicator>}
```

Comment:
Why `!event.availableOnline`?

Reply:
once it's available online, there is a play icon instead

Reply:
or at least there will be - in another PR